src/data/services-manager/autoyast_summary.erb

```diff
@@ -1,16 +1,12 @@
-<%
-  textdomain "services-manager"
```

Comment:
This part is required by `rake pot` otherwise the texts will be missing in the generated POT file.